lib/algora/accounts/user.ex

```diff
@@ -49,6 +50,37 @@ defmodule Algora.Accounts.User do
     end
   end
 
+  def create_or_update_youtube_identity(user, auth) do
+    attrs = %{
+      provider: to_string(auth.provider),
+      provider_id: auth.uid,
+      provider_email: auth.info.email,
+      provider_login: auth.info.email,
+      provider_token: auth.credentials.token,
+      provider_refresh_token: auth.credentials.refresh_token,
```

Comment:
I think Google doesn't provide a refresh token unless we request one:

```elixir
credentials: %Ueberauth.Auth.Credentials{
  token: "...",
  refresh_token: nil,
  token_type: "Bearer",
  expires_at: 1727452355,
}
```

Given that the access token expires in 1 hour, we'll definitely need a refresh token

Reply:
oh okay. I'll look at this too.